CHANGELOG.md

```diff
@@ -45,6 +45,11 @@ Any violations of this scheme are considered to be bugs.
 
 * Your contribution here.
 
+### Miscellaneous
+
+* [#538](https://github.com/hashie/hashie/pull/538): Dropped testing on outdated Rubies, though not necessarily support - [@michaelherold](https://github.com/michaelherold).
```

Comment:
I think you should be explicit in what's dropped, e.g. "Ruby 2.1, 2.2, 2.3 and 2.4 are no longer supported".

I vote to keep them though, unless the build is failing though, or we have another reason to drop the versions, for a little longer. Hashie is fairly low level library and we don't want to force deprecations too broadly, it creates work for everyone.

Reply:
Roger that. I will back it out to just JRuby 9.0 before merging.

Re: Miscellaneous, I am surprised about that. I thought that there was one.